http/jsonapi/runtime/standart_params.go

```diff
@@ -4,24 +4,46 @@
 package runtime
 
 import (
+	"fmt"
 	"net/http"
 	"strconv"
 	"strings"
+
+	"github.com/caarlos0/env"
+	"github.com/pace/bricks/maintenance/log"
 )
 
 // QueryOption is a function that applies an option (like sorting, filter or pagination) to a database query
 type QueryOption func(query Query) Query
 
 // Query based on orm.Query but only the needed function
 // guaranties testability of all of the following functions
+// An Implementation MUST prepare statements to prevent SQL injection
 type Query interface {
 	Offset(n int) Query
 	Limit(n int) Query
 	Order(orders ...string) Query
 	Where(condition string, params ...interface{}) Query
 }
 
-type Mapper func(string) string
+type config struct {
+	MaxPageSize int `env:"MAX_PAGE_SIZE" envDefault:"100"`
+	MinPageSize int `env:"MIN_PAGE_SIZE" envDefault:"1"`
+}
+
+var cfg config
+
+func init() {
+	err := env.Parse(&cfg)
+	if err != nil {
+		log.Fatalf("Failed to parse server environment: %v", err)
```

Comment:
lets be more specific, otherwise we don't understand right from the message, the cause of the issue:

```suggestion
		log.Fatalf("Failed to parse jsonapi params from environment: %v", err)
```